public/js/clients/firefox/events.js

```diff
@@ -59,19 +65,18 @@ function resumed(_, packet) {
 function newSource(_, packet) {
   const { source } = packet;
 
-  if (NEW_SOURCE_IGNORED_URLS.indexOf(source.url) > -1) {
-    return;
-  }
-
-  if (source.introductionType == "debugger eval") {
-    return;
+  source.hidden = false;
+  if (isHidden(source)) {
+    source.url = `evals/console-evauation-${evalIndex++}.js`;
+    source.hidden = true;
```

Comment:
This is the biggest question mark. 

While testing I ran into several places where a source was expected, but wasn't  in the store. e.g. Select source. This changes adds sources to the store, but hides them from store